main.development.js

```diff
@@ -136,59 +53,42 @@ server.use(restify.CORS({
 
 server.listen(5000, function(){
     console.log('%s listening at %s', server.name, server.url);
-})
-
-
-// extract variables from the sent url to localhost 5000
-var usr = 'readonly';
-var psw = 'password';
-var db = 'world';
-var prt = 3308;
-var engine = 'mysql';
-
-console.log('Trying to connect to database ' + db + ' as ' + usr);
-
-const sequelize = new Sequelize(db, usr, psw, {
-    dialect: engine, // 'mysql'|'mariadb'|'sqlite'|'postgres'|'mssql'
-    port:    prt, // or 5432 (for postgres)
 });
 
 mainWindow.webContents.on('did-finish-load', function() {
 
-    console.warn('making request');
-    sequelize.authenticate().then(msg => {
-        console.warn('succcess: ', msg);
-        mainWindow.webContents.send('channel', {success: msg});
+    ipcMain.on('connect', function(event, payload) {
+        var usr = 'root';
+        var psw = 'MyPassword1';
+        var db = 'fake_plotly';
+        var prt = 3306;
+        var engine = 'mysql';
 
-    }).catch(err => {
-        console.warn('failed: ', err);
-        mainWindow.webContents.send('channel', {error: err});
+        console.log('payload: ', payload);
```

Comment:
let's try to start keeping `console.log`s and other temp code out of these branches!